xcm-support/src/currency_adapter.rs

```diff
@@ -38,35 +37,53 @@ impl From<Error> for XcmError {
 ///
 /// If the asset is known, deposit/withdraw will be handled by `MultiCurrency`,
 /// else by `UnknownAsset` if unknown.
-pub struct MultiCurrencyAdapter<MultiCurrency, UnknownAsset, Matcher, AccountIdConverter, AccountId, CurrencyId>(
+pub struct MultiCurrencyAdapter<
+	MultiCurrency,
+	UnknownAsset,
+	Matcher,
+	AccountIdConvert,
```

Comment:
may be `AccountId, AccountIdCovert, CurrencyId, CurrencyIdConvert` to make it more consistent

Reply:
Updated.